### PR TITLE
CORE-5575: Upgrade to Kotlin 1.7.10.

### PR DESCRIPTION
**Pull request by chrisr3**
Upgrade to Kotlin 1.7.10, which is a bug-fix release for Kotlin 1.7.0.

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -7,10 +7,10 @@ cordaProductVersion = 5.0.0
 # NOTE: update this each time this module contains a breaking change
 ## NOTE: currently this is a top level revision, so all API versions will line up, but this could be moved to
 ##   a per module property in which case module versions can change independently.
-cordaApiRevision = 137
+cordaApiRevision = 139
```
Comment (driessamyn) [inline gradle.properties]: Should this be 138?

Comment (chrisr3) [reply gradle.properties]: Not according to `#corda-platform-api-version-lock`.

**Review by driessamyn — APPROVED** [review]
LGTM

**Review by ronanbrowne — APPROVED** [review]
LGTM

**Pull request merged**